docs/network/index.html

```diff
@@ -1181,7 +1181,18 @@ <h4 id="optionsObject">Cluster methods options object</h4>
                     node
                     any way you want. This is also the style object that is provided in the processProperties function
                     for
-                    fine tuning. If undefined, default node options will be used.
+                    fine tuning. If undefined, default node options will be used.<br/><br/>
+                    Default functionality only allows clustering if the cluster contains 2 or more nodes. To allow clustering of single nodes you can use the <code>allowSingleNodeCluster : true</code> property.
+<pre class="prettyprint lang-js">
+var clusterOptions = {
+    joinCondition: function (potentialNodeOptions) {
```

Comment:
"joinCondition" is not always optional. Thats why I chose to leave it in, so it is a complete example, but please confirm you want it removed?